apps/judicial-system/message-handler/src/main.ts

```diff
@@ -3,22 +3,28 @@ import express from 'express'
 import { Logger } from '@nestjs/common'
 import { NestFactory } from '@nestjs/core'
 
-import { AppModule, MessageService } from './app'
+import { LoggingModule } from '@island.is/logging'
+
+import { AppModule, MessageHandlerService } from './app'
 
 async function bootstrap() {
   const port = process.env.PORT || 3366
 
-  const app = await NestFactory.create(AppModule)
+  const app = await NestFactory.create(AppModule, {
+    logger: LoggingModule.createLogger(),
+  })
   app.enableShutdownHooks()
-
   app.use(express.json({ type: ['application/json'] }))
 
+  // Give the app a chance to start up before starting to handle messages
+  await new Promise((resolve) => setTimeout(resolve, 10000))
```

Comment:
Can't this be configured somewhere else? Are there some configuration params that we can pass to the queue/runner?

Reply:
Yes, forgot to reference the config